attestation-agent/kbs_protocol/src/client/mod.rs

```diff
@@ -46,6 +46,9 @@ pub struct KbsClient<T> {
 
     /// token
     pub(crate) token: Option<Token>,
+
+    /// initdata
```

Comment:
I think it'd be better to make the initdata digest or the initdata content more concrete.